src/TodoListItem.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react'
 
-const TodoListItem = (props) => {
+const TodoListItem = ({item:{title}}) => {
```

Comment:
...continued

and then here it could be changed to

```suggestion
const TodoListItem = ({todo:{title}}) => {
```

or, to make things easier in a future lesson where both the `title` and `id` are utilized

```suggestion
const TodoListItem = ({todo}) => {
```

and then you'd just display the title like this `<li>{todo.title}</li>`

Either way works fine as long as you understand what is happening along the way.